app/components/shared/inputs/BaseInput.ts

```diff
@@ -25,6 +26,10 @@ export abstract class BaseInput<TValueType, TMetadataType extends IInputMetadata
     return { required: this.options.required };
   }
 
+  fontFamilyTooltip = $t(
+    'The Google Font to use for the text. Visit http://google.com/fonts to find one! Popular Fonts include: Open Sans, Roboto, Oswald, Lato, and Droid Sans.'
+  );
```

Comment:
This looks like a really specific tooltip.  Does this belong in `BaseInput`?

Reply:
every `type="fontFamily"` input will have it, wasn't really sure where else to put it to ensure it appears on every input